lib/lorina/lorina/diagnostics.hpp

```diff
@@ -109,11 +119,19 @@ class diagnostic_engine
    */
   void emit_custom_diagnostic( diag_id id, std::vector<std::string> const& args ) const;
 
+  /*! \brief Collect all ids and messages
+   *
+   * \param original_diag_ids_args All pairs of the (id, message)
+   */
+  void collect_messages( const std::vector<std::pair<diag_id, std::vector<std::string>>>& original_diag_ids_args );
```

Comment:
```suggestion
  void collect_messages( std::vector<std::pair<diag_id, std::vector<std::string>>> const& original_diag_ids_args );
```
To be consistent with `mockturtle`'s east-`const`

Reply:
Done.